packages/graph/hash_graph/lib/graph/src/store/query/filter.rs

```diff
@@ -129,161 +128,97 @@ impl<'p> Filter<'p, Entity> {
         ])
     }
 
-    /// Creates a `Filter` to search for a specific entity edition, identified by its
-    /// [`EntityVertexId`].
+    /// Creates a `Filter` to search for linked entities where the specified [`EntityId`] identifies
```

Comment:
Should we make it clear that it's the `outgoing linked entities`?
```suggestion
    /// Creates a `Filter` to search for outgoing linked entities where the specified [`EntityId`] identifies
```
(breaks rustfmt)

Reply:
75bc0acc3b6894c1bf282105e1b19bade510c748